packages/rfw/lib/src/flutter/runtime.dart

```diff
@@ -226,6 +244,25 @@ class Runtime extends ChangeNotifier {
     _clearCache();
   }
 
+  /// The widget libraries imported in this [Runtime].
+  ///
+  /// The returned map is an immutable view of the map updated by calls to
+  /// [update] and [clearLibraries].
+  /// 
+  /// The keys are instances [LibraryName] which encode fully qualified library
+  /// names, and the values are the corresponding [WidgetLibrary]s.
+  /// 
+  /// The map may change during the lifetime of the [Runtime], and a new
```

Comment:
That's rather terrifying as an API contract. You get a map that can arbitrarily change out from under you without notification? Should we return a copy instead?

Reply:
Done.